amazon-sqs/5.8/modules/ROOT/pages/amazon-sqs-connector-reference.adoc

```diff
@@ -735,34 +810,43 @@ Sets the value of one or more queue attributes. When you change a queue's attrib
 |Type |String
```

Comment:
Same applies here, String returned by this operation contains ID of the request, shouldn't we mention it here?